photon-lib/src/main/java/org/photonvision/PhotonPoseEstimator.java

```diff
@@ -155,6 +157,24 @@ public void setFieldTags(AprilTagFieldLayout fieldTags) {
         this.fieldTags = fieldTags;
     }
 
+    /**
+     * Get the TargetModel representing the tags being detected. This is used for on-rio multitag.
+     *
+     * <p>By default, this is {@link TargetModel#kAprilTag16h5}.
+     */
+    public TargetModel getTagModel() {
+        return tagModel;
+    }
+
+    /**
+     * Set the TargetModel representing the tags being detected. This is used for on-rio multitag.
+     *
+     * @param tagModel E.g. {@link TargetModel#kAprilTag16h5}.
```

Comment:
I think this link is broken? Not sure, coz it doesn't include the size like we do in TargetModel above

Reply:
This is `photon-targeting`'s `TargetModel`, not `photon-core`'s. We should see if merging these is possible.

Reply:
Yeah totally. Or rename one of them lol